psi4_FFI_to_strain.py

```diff
@@ -0,0 +1,426 @@
+"""
+This module processes gravitational wave data from numerical relativity simulations.
+
+The module reads a directory of ASCII files of various ell values of waveform data, computes phase and amplitude data,
+calculates a minimum data frequency using a quadratic fit of the monotonic phase of  ell2 m2 data,
+and uses a Fast Fourier Transform to compute the second time integral of the waveform, (the strain).
+The module computes a second derivative of the result to check against the original data.
+
+The phase and amplitude, the second integral, and the twice integrated-twice differentiated data is saved to txt files.
+The primary function returns the second integral data as a numpy array with the various ell-values.
+
+Author: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+"""
+
+import sys
+import os
+from typing import List, Tuple
```

Comment:
Please confirm that 
1. `mypy --strict` passes
2. `pylint` passes
3. `pydocstyle` passes
4. doctests pass

Reply:
New commit, mypy --strict passes except at include errors from scipy.optimize
pylint gives 9.8/10 because the primary driving function is too large 
pydocstyle passes